tests/FunctionalJavascript/DefaultContentOwnerTest.php

```diff
@@ -79,4 +79,56 @@ public function testContentOwnerValues() {
     $assert_session->fieldValueEquals('oe_content_content_owner[0][target_id]', '');
   }
 
+  /**
+   * Tests the author values on new nodes.
+   */
+  public function testAuthorsDefaultValues() {
+    // Set the default content owner values.
+    $this->container->get('config.factory')->getEditable('oe_corporate_site_info.settings')
+      ->set('content_owners', [
+        'http://publications.europa.eu/resource/authority/corporate-body/AGRI',
+        'http://publications.europa.eu/resource/authority/corporate-body/BUDG',
+      ])
+      ->save();
+
+    $user = $this->drupalCreateUser([
+      'access content',
+      'create oe_page content',
+      'view published skos concept entities',
+    ]);
+
+    $this->drupalLogin($user);
+
+    $this->drupalGet('/node/add/oe_page');
+
+    $assert_session = $this->assertSession();
+
+    // Assert that Corporate body author values are set
+    // when creating a new node.
+    $this->getSession()->getPage()->pressButton('ief-oe_authors-form-entity-edit-0');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $assert_session->fieldValueEquals('oe_authors[form][inline_entity_form][entities][0][form][oe_skos_reference][0][target_id]', 'Directorate-General for Agriculture and Rural Development (http://publications.europa.eu/resource/authority/corporate-body/AGRI)');
+    $assert_session->fieldValueEquals('oe_authors[form][inline_entity_form][entities][0][form][oe_skos_reference][1][target_id]', 'Directorate-General for Budget (http://publications.europa.eu/resource/authority/corporate-body/BUDG)');
```

Comment:
Let's test that we can save the node correctly and the values are there correctly.